Resources/Textures/_RMC14/Mobs/Customization/human_facial_hair.rsi/meta.json

```diff
@@ -0,0 +1,15 @@
+﻿{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/blob/master/icons/mob/humans/human_facial.dmi",
```

Comment:
Copyright source is incorrect, try using RSI edit.
https://github.com/space-wizards/RSIEdit
https://discord.com/channels/1168210010233376858/1255337028632051792/1280263511234314241